Tests/test_NCBIXML.py

```diff
@@ -1752,12 +1746,11 @@ def test_xml_2222_blastp_001(self):
         handle.close()
 
     def test_xml_2218L_rpsblast_001(self):
-        "Parsing PSI-BLASTP 2.2.18, single query which converges in 3 iterations (xml_2218L_rpsblast_001)"
+        """Parse PSI-BLASTP 2.2.18, single query which converges in 3 iterations (xml_2218L_rpsblast_001)."""
```

Comment:
Here the docstring and comment do not match the method name and filename, which both look like RSP-BLAST.

(This is a pre-existing issue but worth fixing now)